docs/service-introspection-onboarding.md

```diff
@@ -25,101 +48,134 @@ following properties of this storage before proceeding:
-   the tasks in each of the pipelines to access the storage.
-
-   - `ACCOUNT_KEY`: Set this to the access key for your storage account
-   - `ACCOUNT_NAME`: Set this to the name of your storage account
-   - `PARTITION_KEY`: This field can be a distinguishing key that recognizea
-     your source repository in the storage, for eg. in this example, we're using
-     the name of the source repository `hello-bedrock`
-   - `TABLE_NAME`: Set this to the name of the table in your storage account
-     that you prefer to use
-
-   ![](./images/variable_group.png)
-
-   Make sure that you update the pipelines in the following steps to include
-   this variable group, such as below:
-
-   ```yaml
-   variables:
-     - group: <your-variable-group-name>
-   ```
-
-2. To your CI pipeline that runs from the source repository to build the docker
-   image, copy and paste the following task which will update the database for
-   every build that runs from the source repository to show up in Spektate.
-
-   ```yaml
-   - bash: |
-       curl $SCRIPT > script.sh
-       chmod +x ./script.sh
-       tag_name="hello-spektate-$(Build.SourceBranchName)-$(Build.BuildId)"
-       commitId=$(Build.SourceVersion)
-       commitId=$(echo "${commitId:0:7}")
-       ./script.sh $(ACCOUNT_NAME) $(ACCOUNT_KEY) $(TABLE_NAME) $(PARTITION_KEY) p1 $(Build.BuildId) imageTag $tag_name commitId $commitId service $(Build.Repository.Name)
-     displayName: Update manifest pipeline details in CJ db
-     env:
-       SCRIPT: https://raw.githubusercontent.com/catalystcode/spk/master/scripts/update_introspection.sh
-   ```
-
-   Make sure the variable `tag_name` is set to the tag name for the image being
-   built in your docker step.
-
-   **Note**: The earlier in the pipeline you add this task, the earlier it will
-   send data to Spektate. Adding it before the crucial steps is recommended
-   since it will capture details about failures if the important steps fail.
-
-3. To your CD release pipeline (ACR to HLD), add the following lines of code to
-   the end of your last release task (make sure this is not a separate task in
-   the process):
-
-   ```yaml
+**Note:** The Azure storage account is needed to store information about your
+pipelines and services that is displayed by service introspection.
+
+2. Create a table. Follow these
+   [instructions](https://docs.microsoft.com/en-us/azure/storage/tables/table-storage-quickstart-portal).
+
+### Pipelines Configuration
+
+The Bedrock GitOps pipelines need to be configured to start sending data to
+`spk` service introspection. This is done by adding a script snippet in each
+`azure-pipelines.yml` configuration.
+
+#### 1. Configure a variable group
+
+To send data from Azure pipelines to the service introspection storage created
+previously a variable group needs to be configured in Azure DevOps (where the
+pipelines are).
+
+To configure the variable group run:
+
+```
+spk variable-group create
```

Comment:
don't remove the steps to manually create the variable group and the screenshot, in case they would like to create it manually or add it to an existing Variable Group

Reply:
@sarath-p will complete this information. I suspect that it will include the existing steps to create it manually.

Reply:
i think we can eventually automate the manual steps and make both options available, but from this task perspective, we can limit to manual steps.